src/styles/import.css

```diff
@@ -0,0 +1,3 @@
+@import "./reset";
+@import "./stars";
```

Comment:
I'm not sure, but add .css to this and previous import

Reply:
agree